### PR TITLE
REM-706

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
add 2 new fileds to ct-default : 
ct_min_client_CIVAmount 
ct_default_civ_client

### CODE REVIEW

File: src/pages/ct-default/form/CtDefaults.js [hunk 1]
```diff
@@ -112,166 +128,216 @@ const CtDefaults = ({ _labels, access }) => {
...
+                    maxAccess={access}
+                    onChange={e => formik.setFieldValue('ct_min_client_CIVAmount', e.target.value)}
+                    onClear={() => formik.setFieldValue('ct_min_client_CIVAmount', '')}
+                    allowNegative={false}
+                  />
+                </Grid>
+                <Grid item xs={12}>
+                  <ResourceLookup
+                    endpointId={CTCLRepository.CtClientIndividual.snapshot}
+                    parameters={{ _category: 1, _size: 30 }}
```
Comment (NoraBitarSM) [inline src/pages/ct-default/form/CtDefaults.js:318]: snapshot doesn't take _size

File: src/pages/ct-default/index.js [hunk 2]
```diff
@@ -3,7 +3,15 @@ import { ResourceIds } from 'src/resources/ResourceIds'
 import CtDefaults from './form/CtDefaults'
 
 const CtIndex = () => {
-  return <ImmediateWindow datasetId={ResourceIds.CtDefaults} labelKey={'rmd'} Component={CtDefaults} height={700} />
+  return (
+    <ImmediateWindow
+      datasetId={ResourceIds.CtDefaults}
+      labelKey={'rmd'}
+      Component={CtDefaults}
+      height={680}
```
Comment (NoraBitarSM) [inline src/pages/ct-default/index.js:11]: please decrease the height a little bit and increase the width so the lookup labels are visible

**Review by NoraBitarSM — CHANGES_REQUESTED** [review]

File: src/pages/ct-default/form/CtDefaults.js [hunk 3]
```diff
@@ -112,166 +128,216 @@ const CtDefaults = ({ _labels, access }) => {
...
+                <Grid item xs={12}>
+                  <ResourceComboBox
+                    endpointId={MultiCurrencyRepository.RateType.qry}
+                    values={formik.values}
+                    name='ct_credit_eval_ratetype_id'
+                    label={_labels.credit_eval_ratetype}
+                    valueField='recordId'
+                    displayField='name'
+                    onChange={(event, newValue) => {
+                      formik && formik.setFieldValue('ct_credit_eval_ratetype_id', newValue?.recordId || null)
```
Comment (imtithalhalalo) [inline src/pages/ct-default/form/CtDefaults.js:256]: is there a need to check for formik?

File: src/pages/ct-default/form/CtDefaults.js [hunk 4]
```diff
@@ -53,7 +58,11 @@ const CtDefaults = ({ _labels, access }) => {
       ct_credit_purchase_ratetype_id: null,
       ct_credit_eval_ratetype_id: null,
       ct_minOtp_CIVAmount: null,
-      'ct-client-trial-days': null
+      'ct-client-trial-days': null,
+      ct_min_client_CIVAmount: '',
```
Comment (imtithalhalalo) [inline src/pages/ct-default/form/CtDefaults.js:62]: why do you have them as empty strings and not null?

**Review by imtithalhalalo — APPROVED** [review]

**Pull request merged**